plugin/authentication_fido/src/plugin_fido.cc

```diff
@@ -282,6 +286,9 @@ int fido_authenticate(MYSQL_PLUGIN_VIO *vio, MYSQL_SERVER_AUTH_INFO *info) {
   std::vector<unsigned char> decoded(512);
   auto decoded_len = base64_decode(info->auth_string, info->auth_string_length,
                                    decoded.data(), nullptr, 0);
+  if (decoded_len < 65) {
```

Comment:
where 65 comes from ?
Do we have a pre-processor macro or constant defined anywhere for this?

Reply:
Later in the code, we read the data starting with the 64th byte. I increased it to 65 so we also report an error if it's zero length.

Reply:
I added a const for this.